src/data/projects/newrelic-experimental-NewRelicReactNativeModule.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "NewRelicReactNativeModule",
+  "fullName": "newrelic-experimental/NewRelicReactNativeModule",
+  "slug": "newrelic-experimental-NewRelicReactNativeModule",
+  "owner": {
+    "login": "newrelic-experimental",
+    "type": "Organization"
+  },
+  "title": "New Relic React Native Module",
+  "supportUrl": null,
+  "githubUrl": "https://github.com/newrelic-experimental/NewRelicReactNativeModule",
+  "permalink": "https://opensource.newrelic.com/projects/newrelic-experimental/NewRelicReactNativeModule",
+  "defaultBranch": "main",
+  "contributingGuideUrl": null,
+  "iconUrl": "https://github.com/newrelic-experimental/NewRelicReactNativeModule/blob/main/icon.png?raw=true",
+  "shortDescription": "Experimental module for mobile applications built with React Native.",
+  "description": "This module utilizes native New Relic agents to expose the Javascript environment. The New Relic SDKs collect crashes, network traffic, and other information for hybrid apps using native components.",
```

Comment:
@azeazasloper The description is a touch long and ends up getting chopped off on the site, see [here](https://pr-640.d2pwh5nf8rvczf.amplifyapp.com/projects/newrelic-experimental/NewRelicReactNativeModule). Can we shorten this a bit?

Reply:
@devfreddy Fixed!